### PR TITLE
Crowd counting

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@chicco785 could you r? please

File: specs/Transportation/CrowdFlowObserved/doc/spec.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+## Data Model
+
++ `id`: Entity id. 
+    + It shall be `urn:ngsi-ld:CrowdFlowObserved:<identifier>` being `identifier` a unique identifier. 
+
++ `type`: Entity type. 
+    + It shall be equal to `CrowdFlowObserved`.
+    
++ `dateCreated` : Entity's creation timestamp. (`createdAt` in NGSI-LD)
```
Comment (chicco785) [inline specs/Transportation/CrowdFlowObserved/doc/spec.md]: any reason why in NGSI-LD the attribute is different?

File: specs/Transportation/CrowdFlowObserved/doc/spec.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Read-Only. Automatically generated. 
+ 
++ `dateModified` : Last update timestamp of this Entity. (`modifiedAt` in NGSI-LD)
```
Comment (chicco785) [inline specs/Transportation/CrowdFlowObserved/doc/spec.md]: see above

File: specs/Transportation/CrowdFlowObserved/doc/spec.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    + Read-Only. Automatically generated.
+  
++ `source` : A sequence of characters giving the original source of the Entity data as a URL.
+    + Attribute type: [URL](https://schema.org/URL)
+    + Mandatory
+
++ `location` : Location of this crowd flow observation represented by a GeoJSON geometry. 
+    + Attribute type: `geo:json`. (`GeoProperty`)
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory if `refRoadSegment` is not present.
```
Comment (chicco785) [inline specs/Transportation/CrowdFlowObserved/doc/spec.md]: wouldn't be also address ok?

File: specs/Transportation/CrowdFlowObserved/doc/spec.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
++ `description` : Description of this observation.
+    + Normative References: [https://schema.org/description](https://schema.org/description)
+    + Optional
+    
++ `peopleCount` : Total number of people detected during this observation period.
+    + Attribute type: [Number](https://schema.org/Number). Positive integer. 
+    + Optional
+
++ `occupancy` : Fraction of the observation time where a person has been occupying the observed lane.
+    + Attribute type: Property. [Number](https://schema.org/Number) between 0 and 1.
```
Comment (chicco785) [inline specs/Transportation/CrowdFlowObserved/doc/spec.md]: is there a lane concept related to people flow??? what would you define as lane in a walkway?

File: specs/Transportation/CrowdFlowObserved/doc/spec.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
++ `congested` : Flags whether there was a crowd congestion during the observation period in the referred lane. The absence of this attribute means no crowd congestion.  
+    + Attribute type: Property. [Boolean](https://schema.org/Boolean)
+    + Optional
+    
++ `averageHeadwayTime` : Average headway time. Headway time is the time elapsed between two consecutive persons.
+    + Attribute type: Property. [Number](https://schema.org/Number)
+    + Default unit: second (s)
+    + Optional
+    
++ `direction` : Usual direction of travel in the lane referred by this observation with respect to the city center.
```
Comment (chicco785) [inline specs/Transportation/CrowdFlowObserved/doc/spec.md]: this is also weird to me. let's suppose were a talking about a walkway, is there a usual direction for a walkway?

File: specs/Transportation/CrowdFlowObserved/doc/spec.md (hunk 1 above)
Comment (jmcanterafonseca) [reply specs/Transportation/CrowdFlowObserved/doc/spec.md]: standardization-wise ;)

File: specs/Transportation/CrowdFlowObserved/doc/spec.md (hunk 2 above)
Comment (jmcanterafonseca) [reply specs/Transportation/CrowdFlowObserved/doc/spec.md]: standardization-wise

File: specs/Transportation/CrowdFlowObserved/doc/spec.md (hunk 3 above)
Comment (jmcanterafonseca) [reply specs/Transportation/CrowdFlowObserved/doc/spec.md]: I think so, it we take address just to be for instance a street in a city

File: specs/Transportation/CrowdFlowObserved/doc/spec.md (hunk 4 above)
Comment (jmcanterafonseca) [reply specs/Transportation/CrowdFlowObserved/doc/spec.md]: you are right. I think it was a cut&paste issue.

File: specs/Transportation/CrowdFlowObserved/doc/spec.md (hunk 5 above)
Comment (jmcanterafonseca) [reply specs/Transportation/CrowdFlowObserved/doc/spec.md]: you are right. I think it was a cut&paste issue.

**Comment by jmcanterafonseca** [comment]
@chicco785 review comments, addressed, please could you re-check?

Thanks!

**Comment by jmcanterafonseca** [comment]
@chicco785 please could you make a final review so that we can merge?

**Pull request merged**